src/LanguageServer/Impl/Documentation/DocstringConverter.cs

```diff
@@ -87,7 +87,7 @@ private int NextBlockIndent
         private string CurrentLineWithinBlock => CurrentLine.Substring(_blockIndent);
 
         private DocstringConverter(string input) {
-            _builder = new StringBuilder(input.Length);
+            _builder = new StringBuilder(input?.Length ?? 0);
```

Comment:
This constructor is called only from:
```cs
        public static string ToMarkdown(string docstring) {
            if (string.IsNullOrWhiteSpace(docstring)) {
                return string.Empty;
            }

            return new DocstringConverter(docstring).Convert();
        }
```
so it shouldn't be null

Reply:
Probably just missing a merge of #1193.

Reply:
Yes